pages/common/exiftool.md

```diff
@@ -3,9 +3,17 @@
 > Read and write meta information in files.
 > More information: <https://exiftool.org>.
 
+- Print the EXIF metadata for a given file:
+
+`exiftool {{file}}`
+
 - Remove all EXIF metadata from the given files:
 
-`exiftool -All= {{file1 file2 ...}}`
+`exiftool -All= {{file1}} {{file2}} {{file3}}`
```

Comment:
```suggestion
`exiftool -All= {{file1 file2 ...}}`
```
Revert this change. As this is what we use as a placeholder for multiple files. Check out our [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) to know more.

https://github.com/tldr-pages/tldr/blob/d6673b79b546780e485fddd3c4287b7cbc4177a8/contributing-guides/style-guide.md?plain=1#L148